Jenkinsfile

```diff
@@ -146,32 +102,48 @@ pipeline {
 
       post {
         always {
-          archiveArtifacts 'buildinfo.json'
+          archiveArtifacts "buildinfo.json"
         }
       }
     }
 
-    stage('Release Version'){
+    stage("Run Enterprise Pipeline") {
       when {
         expression {
-          release_branches.contains(BRANCH_NAME) && params.CREATE_RELEASE == true
+          master_branches.contains(BRANCH_NAME)
         }
       }
-
       steps {
-        withCredentials([
-            string(credentialsId: '3f0dbb48-de33-431f-b91c-2366d2f0e1cf',variable: 'AWS_ACCESS_KEY_ID'),
-            string(credentialsId: 'f585ec9a-3c38-4f67-8bdb-79e5d4761937',variable: 'AWS_SECRET_ACCESS_KEY'),
-            usernamePassword(credentialsId: 'a7ac7f84-64ea-4483-8e66-bb204484e58f', passwordVariable: 'GIT_PASSWORD', usernameVariable: 'GIT_USER')
-        ]) {
-          sh "git config --global user.email $GIT_USER@users.noreply.github.com"
-          sh "git config --global user.name 'MesosphereCI Robot'"
-          sh "git config credential.helper 'cache --timeout=300'"
-
-          sh "CREATE_RELEASE=1 PUSH_RELEASE=1 ./scripts/ci/bump-version"
+        build job: "frontend/dcos-ui-ee-pipeline/" + env.BRANCH_NAME.replaceAll("/", "%2F")
+      }
+    }
+  }
 
-          sh "./scripts/ci/release-version"
-        }
+  post {
```

Comment:
These changes are not really in the same scope as "Removing release parts". Can we please add them in a separate commit / PR / Ticket and not scope creep here? Don't get me wrong, I like the change in general, but we are not showing our real performance if we do stuff like this "just on the side". Sorry for being picky here 🙈

Reply:
This PR is about wrapping up my work done in the `feature/ci` branch. not _only_ about removing release stuff. so no scope creep here 👍

Reply:
Okay, I see the slack send in the Jira body

Reply:
yeah, because I added it :trollface:

Reply:
Is this how we work now? 😕

Reply:
I forgot it in the first place, you've been picky, I've added it… I guess thats a "yes"?

Reply:
Detail and scope are very different things, we should make clear we don't mix them here

Reply:
Thats what I'm talking about. The *scope* of this ticket is to wrap up the work done on `feature/ci` - while creating this ticket, I forgot to add a *detail* and might have written a bad description, but there was no scope creep.

Reply:
Okay, now I got you 👍 So the main problem was the ambiguity of the ticket itself, because "cleaning up" is very vague, got it 👍 Let's make sure we make clearer tickets next time, I will have an extra focused view on that next sprints :)

Reply:
I'm more then happy about any help I can get with cleaning up our old Epics and moving them under https://jira.mesosphere.com/browse/PRODUCT-1000 👍 just ping me, if you want to help.